src/app/utils.ts

```diff
@@ -5,24 +5,24 @@ import jwt from 'jsonwebtoken'
 
 export const UPLOAD_DIR = path.resolve(os.tmpdir(), 'public/uploads')
 
-export const createUploadDirIfNotExists = async () => {
+export const createUploadDirIfNotExists = () => {
     if (!fs.existsSync(UPLOAD_DIR)) {
         fs.mkdirSync(UPLOAD_DIR, { recursive: true })
     }
 }
 
 export const saveFile = async (file: Blob, runId: string) => {
-    await createUploadDirIfNotExists()
+    createUploadDirIfNotExists()
     const buffer = Buffer.from(await file.arrayBuffer())
     try {
         fs.writeFileSync(path.resolve(UPLOAD_DIR, runId), buffer)
-    } catch (err) {
+    } /* v8 ignore next 3 */ catch (err) {
```

Comment:
Maybe per our meeting last week this should crash the program instead of quietly failing

Reply:
yes good catch - i hadn't gotten around to this yet after that meeting 👍